### PR TITLE
Handle UnicodeEncodeError of convert unicode string to ascii string

### PR DESCRIPTION
**Pull request by shawntian90**
`docker_username` and `docker_password` may be unicode string.

### CODE REVIEW

**Comment by hao1939** [comment]
Hi @shawntian90 ,

Could you share the background of the PR?
Are you using unicode characters in the configuration file?

**Comment by shawntian90** [comment]
@hao1939 

I wanna to enable my private docker registry, which username is unicode string, such as "容器管理员".

There will be an exception `UnicodeEncodeError` when run commands like `python paictl.py` inside dev-box.

**Comment by xudifsd** [comment]
similar to #809 

I suggest change all `with open("foo") as f` to `with `[`codecs.open`](https://docs.python.org/2/library/codecs.html#codecs.open)`("foo", "utf-8")` should also change output method too.

It's better to search keyword `open` to find input/output calls.

File: pai-management/paictl.py [hunk 1]
```diff
@@ -87,12 +87,21 @@ def login_docker_registry(docker_registry, docker_username, docker_password):
     logger.info("docker registry login successfully")
 
 
+def to_str(value):
```
Comment (ydye) [inline pai-management/paictl.py]: Can you move this function to the common module?

**Comment by shawntian90** [comment]
@xudifsd @ydye 
I will make a patch  to all `open` statement and read configuration file via `codecs.open` this week.

**Pull request closed without merging**